app/bundles/EmailBundle/Entity/StatRepository.php

```diff
@@ -116,6 +143,8 @@ public function getSentEmailToContactData(
         }
 
         $q->setMaxResults($limit);
+        $q->groupBy('s.id');
```

Comment:
Just note: This goes against MySQL full group by

Reply:
true, must be in the select too.